Sources_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -126,6 +126,13 @@ extension ChannelDTO {
         request.predicate = NSPredicate(format: "ANY queries.filterHash == %@", query.filter.filterHash)
         return request
     }
+
+    static var newChannelsFetchRequest: NSFetchRequest<ChannelDTO> {
```

Comment:
I'd suggest renaming this to `channelWithoutQueryFetchRequest`. It's possible to have channels without queries. It doesn't mean they're always new.